### ISSUE TITLE
Chatfilter bypass not working

### ISSUE THREAD

**Issue by dkalaxdk** [Zrips/CMI#4985]
**Description of issue:**
When having given a user `cmi.chatfilter.bypass.swearing` the bypass has no effect, and the user word is still swapped with "dude".

---

**CONFIG SECTION:**

ChatFilter:

    Enabled: true

    Deny:

      ipBlock:

        # If not set to true, this filter will not be used

        Enabled: true

        # Defines filter group and defines required permission node to bypass this filter: cmi.chatfilter.bypass.[groupName]

        Group: Advertising

        # When set to true, each time player triggers filter, console will receive information about who triggered it, which filter and with what message

        InformConsole: true

        # Regex expression to filter by. How to use regex https://regexone.com/

        Regex:

        - '[a-zA-Z0-9\-\.]+\s?(\.|dot|\(dot\)|-|;|:|,|_|\/)\s?([a-zA-Z]{2}|aero|asia|biz|cat|com|coop|edu|gov|info|int|jobs|mil|mobi|museum|name|net|org|pro|tel|travel)\b'

        - \b[0-9]{1,3}(\.|dot|\(dot\)|-|;|:|,|(\W|\d|_)*\s)+[0-9]{1,3}(\.|dot|\(dot\)|-|;|:|,|(\W|\d|_)*\s)+[0-9]{1,3}(\.|dot|\(dot\)|-|;|:|,|(\W|\d|_)*\s)+[0-9]{1,3}\b

        # With what we need to replace word, if not defined found expression will not be changed

        ReplaceWith: ''

        # possible: none, others, all

        # Where 'none' means everyone will receive this message

        # 'others' means that sender will get message but not other players, this is usefull to prevent advertising and silently block it

        # 'all' means that no one will receive sent message

        BlockType: others

        # When enabled rule will be applied to private messages

        includePrivateMessages: true

        # Players with cmi.chatfilter.inform permission will receive defined message when rule is broken

        msgToStaff: '&4!&6[playerName] &4advertising with: &r[message]'

        # List of commands to perform when rule is broken. Use [senderName] to include message sender name. Supports global variables same as locale file

        Commands: []

      swearing1:

        Enabled: true

        Group: Swearing

        InformConsole: true

        Regex:

        - \bass\b|\basshole

        ReplaceWith: dude

        BlockType: none

        includePrivateMessages: true

        msgToStaff: '&4!&6[playerName] &4swearing'

        Commands:

        - msg [senderName] &eDon't swear!

---

**Cmi Version (using`/cmi version`):** 
8.8.0.7
**Server Type (Spigot/Paperspigot/etc):** 
PaperSpigot
**Server Version (using `/ver`):** 
git-Paper-416 (MC:1.16.4)

**Comment by borjxrroyo** [comment]
I would also like to suggest filtering/seeing messages from books, signs and anvils.

- /bookspy
-  /anvilspy

**Comment by dkalaxdk** [comment]
> I would also like to suggest filtering/seeing messages from books, signs and anvils.
> 
> * /bookspy
> * /anvilspy

That might be related to a different issue, as this ain't directly related to /socialspy.
So it might be a better idea to create a feature request on this instead @borjxrroyo

**Comment by borjxrroyo** [comment]
Yep, I've done it a time ago but I think it was forgotten, so I would like to "bump" it :)

**Comment by CloudeLecaw** [comment]
I can agree this issue. Our Staff also cant bypass the swearfilter anymore even they have the correct bypass permission.
This also happens with out created filters
The issue still exists in latest CMI Version.

**Comment by CloudeLecaw** [comment]
Up..

**Comment by CloudeLecaw** [comment]
Up..

**Comment by CloudeLecaw** [comment]
fixed in last updaqte - can be closes
